src/run.c

```diff
@@ -461,18 +533,19 @@ Port io_get_time(Net* net, Book* book, Port argm) {
   u32 lps = 0;
   u32 loc = node_alloc_1(net, tm[0], &lps);
   node_create(net, loc, new_pair(new_port(NUM, new_u24(time_hi)), new_port(NUM, new_u24(time_lo))));
-  // Return the encoded time
+
   return new_port(CON, loc);
 }
 
 // Sleeps.
 // `argm` is a tuple (CON node) of the high and low
 // 24 bits for a 48-bit duration in nanoseconds.
+// Returns: Result<*, *>
```

Comment:
Since the only error case here is a type error, I think its fine not to return a Result. This is fine, but I'd just unwrap the value in Bend since it should never fail in a correct program

Reply:
We discussed returning a result here, since `call` will return a `Result<T, IOError<E>>`, as the name or args may be invalid when passed to `call`.